onnx/backend/test/cpp/driver/test_driver.h

```diff
@@ -0,0 +1,79 @@
+#include <iostream>
+#include <queue>
+#include <string>
+#include <cstdlib>
+#include <cstdio>
+
+#include "onnx/onnx_pb.h"
+#include "onnx/proto_utils.h"
+#include "onnx/onnxifi_loader.h"
+#include "onnxifi.h"
+
+#include "gtest/gtest.h"
+
+struct TestData{
+	std::vector<std::string> input_filenames_;
+	std::vector<std::string> output_filenames_;
+
+	TestData(){}
+
+	TestData(const std::vector<std::string> &input_filenames, const std::vector<std::string> &output_filenames){
+		input_filenames_ = input_filenames;
+		output_filenames_ = output_filenames;
+	}
+};
+struct TestCase{
+	TestCase(
+		const std::string& model_filename,
+		const std::string& model_dirname,
+		const std::vector<TestData>& test_data){
+
+		model_filename_ = model_filename;
+		model_dirname_ = model_dirname;
+		test_data_ = test_data;
+	}
+
+	TestCase(){}
+
+	std::string model_filename_;
+	std::string model_dirname_;
+	std::vector<TestData> test_data_;
+};
+
+struct ProtoTestData{
+	std::vector<std::string> raw_inputs_;
+	std::vector<std::string> raw_outputs_;
+	std::vector<onnx::TensorProto> inputs_;
+	std::vector<onnx::TensorProto> outputs_;
+};
+
+struct ProtoTestCase{
+	std::string raw_model_;
```

Comment:
This seems to be a temporary variable to me, because its purpose is to create the `model_`.